e2etest/bgp_test.go

```diff
@@ -58,22 +59,20 @@ var _ = ginkgo.Describe("BGP", func() {
 		cs = f.ClientSet
 
 		var err error
-		// If the FRR container is not running, start it on the local host.
-		if skipDockerCmd {
-			if len(frrTestConfigDir) == 0 {
-				framework.Fail("Missing FRR config directory.")
-			}
+		frrTestConfigDir, err = ioutil.TempDir("", "frr-conf")
+		framework.ExpectNoError(err)
+		err = frr.StartContainer(frrContainer, frrTestConfigDir, containerNetwork, containerRuntime)
+		framework.ExpectNoError(err)
+
+		if containerNetwork == "host" {
```

Comment:
Initialize them with the host and override them if network != host, it will avoid the else.